mirage/seed_image/catalog_seed_image.py

```diff
@@ -3076,22 +3080,27 @@ def make_galaxy_image(self, file):
 
             # First create the galaxy
             stamp = self.create_galaxy(entry['radius'], entry['ellipticity'], entry['sersic_index'],
-                                       xposang*np.pi/180., entry['counts_per_frame_e'])
+                                       xposang*np.pi/180., entry['countrate_e/s'])
 
             # If the stamp image is smaller than the PSF in either
             # dimension, embed the stamp in an array that matches
             # the psf size. This is so the upcoming convolution will
             # produce an output that includes the wings of the PSF
             galdims = stamp.shape
 
-            # *******OR SHOULD WE ALWAYS CONVOLVE WITH A LARGER PSF? IF PSF IS SMALL,
-            # THIS WILL ARTIFICIALLY PUSH SIGNAL TOWARDS THE CORE OF THE GALAXY, I THINK.
```

Comment:
This comment here (which is removed in this PR) is correct I think.  Using the smaller PSF normalized to 1 will keep more light near the core of the extended source, compared to the more rigorous approach that uses the full convolution including the wings. Whether this is a problem or not will depend on the relative sizes of the photometry aperture versus the extended source...

Reply:
Hmm. Maybe I'll leave this comment in the code, as a warning in case it is found to be a problem.